test/controllers/organizations/staff/external_form_upload_controller_test.rb

```diff
@@ -0,0 +1,28 @@
+require "test_helper"
+
+module Organizations
+  module Staff
+    class ExternalFormUploadControllerTest < ActionDispatch::IntegrationTest
+      setup do
+        file = fixture_file_upload("google_form_sample.csv", "text/csv")
+        @params = {files: [file]}
+        user = create(:admin)
```

Comment:
I should have mentioned this earlier but technically we are creating user objects here not person objects. This might confuse someone in the future.